### ISSUE TITLE
Add Scroll to Bungee exchange

### ISSUE THREAD

**Issue by darkman5696** [scroll-tech/contribute-to-scroll#953]
### Name of the project

Bungee

### Project type

Library

### Site of the project

https://www.bungee.exchange/

### Github repository of the project

https://github.com/SocketDotTech/plugin

### Networks

Scroll Mainnet

### Additional context

Bungee is one of the most popular Bridges for users with its simple to use user interface and extremely competitive fees.
Bungee uses Socket to facilitate its bridging to ensure a quick, cheap and seamless bridging experience for its customers.

**Comment by wcb0alpha** [comment]
I agree with this proposal as I use Bungee most days and would like to cross-chain swap into Scroll
